### PR TITLE
[17.0][MIG]account_fiscal_position_autodetect_optional_vies: Migration to 17.0

### PR DESCRIPTION
**Pull request by Tisho99**
The optional vies field [is now part of the core of Odoo](https://github.com/odoo/odoo/blob/56666f8f7858fcbcce466d2240135b35509d2d96/addons/base_vat/models/res_partner.py#L94-L99). So, changes have been done assuming base_vat_optional_vies [is not going to be migrated to 17.0 ](https://github.com/OCA/account-financial-tools/issues/1774)

### CODE REVIEW

**Comment by Tisho99** [comment]
@ValentinVinagre

**Comment by ValentinVinagre** [comment]
@Jesarregui @HaraldPanten could u review please?

File: account_fiscal_position_autodetect_optional_vies/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[build-system]
```
Comment (ValentinVinagre) [inline account_fiscal_position_autodetect_optional_vies/pyproject.toml:1]: Is this file necessary?

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Comment by HaraldPanten** [comment]
/ocabot migration account_fiscal_position_autodetect_optional_vies
